src/main/java/net/imglib2/Localizable.java

```diff
@@ -99,7 +104,12 @@ default int getIntPosition( final int d )
 	/**
 	 * Allocate and return a {@link Point} containing the localizable's
 	 * position.
-	 * 
+	 *
+	 * Please note that his method allocates a new {@link Point} each time
+	 * which introduces notable overhead in both compute and memory.
+	 * If you query it frequently, you should allocate a dedicated array
+	 * first and reuse it with {@link #localize(long[])}.
```

Comment:
change doc to:
first and reuse it with{@link #localize(Positionable)}.

Reply:
This method does not exist.

Reply:
Added it.